### ISSUE TITLE
Fails to build on MacOS Ventura (Intel)

### ISSUE THREAD

**Issue by iMichka** [samphippen/rubyfmt#440]
* Ruby version: ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.x86_64-darwin22]
* Rubyfmt git sha: (if unsure, redownload the latest version and run against that): 0.8.1, 266b47b4666106e579626b1823100ccedf5cbbc3

The project fails to build on macOS Ventura, with system ruby.

```
  running: "clang" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "--target=x86_64-apple-darwin" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/include" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/arm64-darwin20" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/arm64-darwin21" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/arm64-darwin22" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/x86_64-darwin21" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/x86_64-darwin20" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/x86_64-darwin19" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/x86_64-darwin18" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/x86_64-linux" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/x64-mswin64_140" "-I" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/.ext/include/i386-mswin32_140" "-Wall" "-Wextra" "-o" "/private/tmp/rubyfmt-20230715-91865-dg4lkz/target/release/build/rubyfmt-9e36bebbc31b7a1b/out/src/rubyfmt.o" "-c" "src/rubyfmt.c"
  cargo:warning=In file included from src/rubyfmt.c:1:
  cargo:warning=In file included from /private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/include/ruby.h:33:
  cargo:warning=/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/include/ruby/ruby.h:24:10: fatal error: 'ruby/config.h' file not found
  cargo:warning=#include "ruby/config.h"
  cargo:warning=         ^~~~~~~~~~~~~~~
  cargo:warning=/private/tmp/rubyfmt-20230715-91865-dg4lkz/librubyfmt/ruby_checkout/include/ruby/ruby.h:24:10: note: did not find header 'config.h' in framework 'ruby' (loaded from '/Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/System/Library/Frameworks')
  cargo:warning=1 error generated.
  exit status: 1
```

From https://github.com/Homebrew/homebrew-core/actions/runs/5558372748/jobs/10153387768.

We are trying to build this package in Homebrew. Can you have a look. If more details are necessary I would be glad to provide them.

**Comment by chenrui333** [comment]
Should be good now with [this PR](https://github.com/Homebrew/homebrew-core/pull/137470) merge

**Comment by reese** [comment]
Sounds like this is resolved (since I see 0.10.0 on [homebrew](https://formulae.brew.sh/formula/rubyfmt#default)), but feel free to reopen if you're still running into issues.
